brushfire-core/src/main/scala/com/stripe/brushfire/Brushfire.scala

```diff
@@ -44,6 +44,9 @@ trait Splitter[V, T] {
 /** Candidate split for a tree node */
 case class Split[V, T](predicate: Predicate[V], leftDistribution: T, rightDistribution: T) {
 
+  def map[U](f: V => U): Split[U, T] =
+    Split(predicate.map(f), leftDistribution, rightDistribution)
```

Comment:
I think there is a `mapPredicates` or something function on `AnnotatedTree` that does this manually - may be nice to rewrite it to use this `map` instead.